tests/clone-populator_test.go

```diff
@@ -309,6 +309,9 @@ var _ = Describe("Clone Populator tests", func() {
 			if !f.IsSnapshotStorageClassAvailable() {
 				Skip("Clone from volumesnapshot does not work without snapshot capable storage")
 			}
+			if utils.DefaultStorageClassCsiDriver == nil {
```

Comment:
instead of putting it here and in the beforeEach of the clone from pvc it can be in one BeforeEach at the beginning of all the clone tests